### PR TITLE
aos: Add meta-aos layer for prod-aos product

### PR DESCRIPTION
**Pull request by Eabramov84**
Signed-off-by: Yevgen Abramov <Yevgen_Abramov@epam.com>

### CODE REVIEW

**Comment by al1img** [comment]
Reviewed-by: Oleksandr Grytsov `<oleksandr_grytsov@epam.com>`

**Review by al1img — APPROVED** [review]

File: prod-aos.xml [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
 <manifest>
     <remote name="github" fetch="git://github.com" />
+    <remote name="epam" fetch="ssh://git@gitpct.epam.com" />
 
     <default remote="github" sync-j="10" sync-c="true" />
 
-    <project name="xen-troops/meta-xt-prod-aos" path="meta-xt-prod-aos" upstream="master" revision="master" />
-    <project name="xen-troops/meta-xt-images" path="meta-xt-images" upstream="master" revision="master" />
-    <project name="xen-troops/xt-distro" path="xt-distro" upstream="master" revision="master" />
+    <project remote="github" name="xen-troops/meta-xt-prod-aos" path="meta-xt-prod-aos" upstream="master" revision="master" />
```
Comment (andr2000) [inline prod-aos.xml]: you don't need to put github here as it is already the default remote, so this change is not relevant

**Comment by andr2000** [comment]
It would be great if you add the exact reason why you add the additional meta layer on top

**Comment by iusyk** [comment]
> It would be great if you add the exact reason why you add the additional meta layer on top

+1

File: prod-aos.xml (hunk 1 above)
Comment (Eabramov84) [reply prod-aos.xml]: accept. Will be removed

**Review by iusyk — APPROVED** [review]
Reviewed-by: Ihor Usyk <ihor_usyk@epam.com>

**Comment by andr2000** [comment]
`Reviewed-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>`

**Pull request merged**